vSphere fetchVM fails lint

[user@host machine]$ make test
0 adds, 0 deletions; nothing to validate! :)
go test  -test.short -tags "" github.com/docker/machine github.com/docker/machine/commands github.com/docker/machine/commands/commandstest github.com/docker/machine/commands/mcndirs github.com/docker/machine/drivers/amazonec2 github.com/docker/machine/drivers/azure github.com/docker/machine/drivers/azure/azureutil github.com/docker/machine/drivers/azure/logutil github.com/docker/machine/drivers/digitalocean github.com/docker/machine/drivers/driverutil github.com/docker/machine/drivers/errdriver github.com/docker/machine/drivers/exoscale github.com/docker/machine/drivers/fakedriver github.com/docker/machine/drivers/generic github.com/docker/machine/drivers/google github.com/docker/machine/drivers/hyperv github.com/docker/machine/drivers/none github.com/docker/machine/drivers/openstack github.com/docker/machine/drivers/rackspace github.com/docker/machine/drivers/softlayer github.com/docker/machine/drivers/virtualbox github.com/docker/machine/drivers/vmwarefusion github.com/docker/machine/drivers/vmwarevcloudair github.com/docker/machine/drivers/vmwarevsphere github.com/docker/machine/its github.com/docker/machine/its/cli github.com/docker/machine/its/thirdparty github.com/docker/machine/libmachine github.com/docker/machine/libmachine/auth github.com/docker/machine/libmachine/cert github.com/docker/machine/libmachine/check github.com/docker/machine/libmachine/crashreport github.com/docker/machine/libmachine/drivers github.com/docker/machine/libmachine/drivers/plugin github.com/docker/machine/libmachine/drivers/plugin/localbinary github.com/docker/machine/libmachine/drivers/rpc github.com/docker/machine/libmachine/engine github.com/docker/machine/libmachine/examples github.com/docker/machine/libmachine/host github.com/docker/machine/libmachine/hosttest github.com/docker/machine/libmachine/libmachinetest github.com/docker/machine/libmachine/log github.com/docker/machine/libmachine/mcndockerclient github.com/docker/machine/libmachine/mcnerror github.com/docker/machine/libmachine/mcnflag github.com/docker/machine/libmachine/mcnutils github.com/docker/machine/libmachine/persist github.com/docker/machine/libmachine/persist/persisttest github.com/docker/machine/libmachine/provider github.com/docker/machine/libmachine/provision github.com/docker/machine/libmachine/provision/pkgaction github.com/docker/machine/libmachine/provision/provisiontest github.com/docker/machine/libmachine/provision/serviceaction github.com/docker/machine/libmachine/shell github.com/docker/machine/libmachine/ssh github.com/docker/machine/libmachine/ssh/sshtest github.com/docker/machine/libmachine/state github.com/docker/machine/libmachine/swarm github.com/docker/machine/libmachine/version github.com/docker/machine/version
?   	github.com/docker/machine	[no test files]
ok  	github.com/docker/machine/commands	0.619s
?   	github.com/docker/machine/commands/commandstest	[no test files]
ok  	github.com/docker/machine/commands/mcndirs	0.004s
ok  	github.com/docker/machine/drivers/amazonec2	0.021s
ok  	github.com/docker/machine/drivers/azure	0.007s
?   	github.com/docker/machine/drivers/azure/azureutil	[no test files]
?   	github.com/docker/machine/drivers/azure/logutil	[no test files]
ok  	github.com/docker/machine/drivers/digitalocean	0.025s
ok  	github.com/docker/machine/drivers/driverutil	0.026s
?   	github.com/docker/machine/drivers/errdriver	[no test files]
ok  	github.com/docker/machine/drivers/exoscale	0.006s
?   	github.com/docker/machine/drivers/fakedriver	[no test files]
ok  	github.com/docker/machine/drivers/generic	0.007s
ok  	github.com/docker/machine/drivers/google	0.010s
ok  	github.com/docker/machine/drivers/hyperv	0.009s
?   	github.com/docker/machine/drivers/none	[no test files]
ok  	github.com/docker/machine/drivers/openstack	0.005s
ok  	github.com/docker/machine/drivers/rackspace	0.035s
ok  	github.com/docker/machine/drivers/softlayer	0.026s
ok  	github.com/docker/machine/drivers/virtualbox	3.543s
?   	github.com/docker/machine/drivers/vmwarefusion	[no test files]
ok  	github.com/docker/machine/drivers/vmwarevcloudair	0.005s
ok  	github.com/docker/machine/drivers/vmwarevsphere	0.010s
?   	github.com/docker/machine/its	[no test files]
ok  	github.com/docker/machine/its/cli	11.153s
ok  	github.com/docker/machine/its/thirdparty	0.003s
?   	github.com/docker/machine/libmachine	[no test files]
?   	github.com/docker/machine/libmachine/auth	[no test files]
ok  	github.com/docker/machine/libmachine/cert	0.878s
ok  	github.com/docker/machine/libmachine/check	0.006s
ok  	github.com/docker/machine/libmachine/crashreport	0.005s
ok  	github.com/docker/machine/libmachine/drivers	0.005s
?   	github.com/docker/machine/libmachine/drivers/plugin	[no test files]
ok  	github.com/docker/machine/libmachine/drivers/plugin/localbinary	0.015s
ok  	github.com/docker/machine/libmachine/drivers/rpc	0.006s
?   	github.com/docker/machine/libmachine/engine	[no test files]
?   	github.com/docker/machine/libmachine/examples	[no test files]
ok  	github.com/docker/machine/libmachine/host	0.007s
?   	github.com/docker/machine/libmachine/hosttest	[no test files]
?   	github.com/docker/machine/libmachine/libmachinetest	[no test files]
ok  	github.com/docker/machine/libmachine/log	0.026s
?   	github.com/docker/machine/libmachine/mcndockerclient	[no test files]
?   	github.com/docker/machine/libmachine/mcnerror	[no test files]
?   	github.com/docker/machine/libmachine/mcnflag	[no test files]
ok  	github.com/docker/machine/libmachine/mcnutils	0.012s
ok  	github.com/docker/machine/libmachine/persist	0.043s
ok  	github.com/docker/machine/libmachine/persist/persisttest	0.027s
?   	github.com/docker/machine/libmachine/provider	[no test files]
ok  	github.com/docker/machine/libmachine/provision	0.011s
ok  	github.com/docker/machine/libmachine/provision/pkgaction	0.003s
ok  	github.com/docker/machine/libmachine/provision/provisiontest	0.025s
ok  	github.com/docker/machine/libmachine/provision/serviceaction	0.016s
ok  	github.com/docker/machine/libmachine/shell	0.009s
ok  	github.com/docker/machine/libmachine/ssh	1.010s
?   	github.com/docker/machine/libmachine/ssh/sshtest	[no test files]
ok  	github.com/docker/machine/libmachine/state	0.003s
?   	github.com/docker/machine/libmachine/swarm	[no test files]
?   	github.com/docker/machine/libmachine/version	[no test files]
?   	github.com/docker/machine/version	[no test files]
drivers/vmwarevsphere/vsphere.go:852:1: context.Context should be the first parameter of a function
make: *** [lint] Error 1

--- a/drivers/vmwarevsphere/vsphere.go
+++ b/drivers/vmwarevsphere/vsphere.go
@@ -236,7 +236,7 @@ func (d *Driver) GetIP() (string, error) {
 	}
 	defer c.Logout(ctx)
 
-	vm, err := d.fetchVM(c, ctx, d.MachineName)
+	vm, err := d.fetchVM(ctx, c, d.MachineName)
 	if err != nil {
 		return "", err
 	}
@@ -279,7 +279,7 @@ func (d *Driver) GetState() (state.State, error) {
 	}
 	defer c.Logout(ctx)
 
-	vm, err := d.fetchVM(c, ctx, d.MachineName)
+	vm, err := d.fetchVM(ctx, c, d.MachineName)
 	if err != nil {
 		return state.None, err
 	}
@@ -592,7 +592,7 @@ func (d *Driver) Start() error {
 		}
 		defer c.Logout(ctx)
 
-		vm, err := d.fetchVM(c, ctx, d.MachineName)
+		vm, err := d.fetchVM(ctx, c, d.MachineName)
 		if err != nil {
 			return err
 		}
@@ -625,7 +625,7 @@ func (d *Driver) Stop() error {
 	}
 	defer c.Logout(ctx)
 
-	vm, err := d.fetchVM(c, ctx, d.MachineName)
+	vm, err := d.fetchVM(ctx, c, d.MachineName)
 	if err != nil {
 		return err
 	}
@@ -681,7 +681,7 @@ func (d *Driver) Kill() error {
 	}
 	defer c.Logout(ctx)
 
-	vm, err := d.fetchVM(c, ctx, d.MachineName)
+	vm, err := d.fetchVM(ctx, c, d.MachineName)
 	if err != nil {
 		return err
 	}
@@ -750,7 +750,7 @@ func (d *Driver) Remove() error {
 		}
 	}
 
-	vm, err := d.fetchVM(c, ctx, d.MachineName)
+	vm, err := d.fetchVM(ctx, c, d.MachineName)
 	if err != nil {
 		return err
 	}
@@ -849,7 +849,7 @@ func (d *Driver) vsphereLogin(ctx context.Context) (*govmomi.Client, error) {
 	return c, nil
 }
 
-func (d *Driver) fetchVM(c *govmomi.Client, ctx context.Context, vmname string) (*object.VirtualMachine, error) {
+func (d *Driver) fetchVM(ctx context.Context, c *govmomi.Client, vmname string) (*object.VirtualMachine, error) {
 
 	// Create a new finder
 	f := find.NewFinder(c.Client, true)

